README.md

```diff
@@ -39,7 +39,7 @@ By default, dnsdist is installed from the software repositories configured on th
       dnsdist_install_repo: "{{ dnsdist_powerdns_repo_13 }}"
 ```
 
-The examples above, show how to install dnsdist from the official PowerDNS repositories
+The examples above, show how to install DNSdist from the official PowerDNS repositories
```

Comment:
why this change?

Reply:
just to have DNSdist written in same way in the readme file, at this moment it use two ways: DNSdist and dnsdist. I think the right one would be dnsdist, as it's opensource roles. I'll do a another MR later for this.

Reply:
:+1: